cmake/FindGarfieldOld.cmake

```diff
@@ -15,14 +15,17 @@
 set(Garfield_DIR $ENV{GARFIELD_HOME})
 
 find_path(Garfield_INCLUDE_DIRS Sensor.hh
-        HINTS ${Garfield_DIR}/include/ ${Garfield_DIR}/include/Garfield ${Garfield_INC_DIR}
-        ${Garfield_DIR}/Include)
+        HINTS ${Garfield_DIR}/Include/ ${Garfield_DIR}/Include/Garfield
+        ${Garfield_DIR}/include/ ${Garfield_DIR}/include/Garfield)
+
+find_path(Garfield_INCLUDE_Heed_DIRS HeedChamber.hh
+        HINTS ${Garfield_DIR}/Heed/)
 
 #message(STATUS Garfield_INCLUDE_DIRS ${Garfield_INCLUDE_DIRS})
 
 find_library(Garfield_LIBRARIES NAMES libGarfield.so Garfield
         HINTS ${Garfield_DIR}/lib ${Garfield_LIB_DIR}
-        $ENV{GARFIELD_HOME}/lib)
+        $ENV{GARFIELD_DIR}/lib)
```

Comment:
I believe this line is not needed and it was actually removed, I don't understand how it end up in this PR.

As fas as I remember some commits were cherry-pick from this PR to master. Perhaps something when wrong in between.

Reply:
This branch is behind master, perhaps it will be fixed once master is merged in this branch. Highly recommended.

<img width="1364" alt="Screenshot 2022-04-20 at 21 35 51" src="https://user-images.githubusercontent.com/12447509/164309202-01869f76-fefd-42c3-832e-a7ab36a79f47.png">

Reply:
I guess thats why "outdated" appears next to the file name